runtime/testruntime/testruntime.go

```diff
@@ -88,9 +89,15 @@ type InstanceOptions struct {
 func NewInstanceWithOptions(t TestingT, opts InstanceOptions) (*runtime.Runtime, string) {
 	rt := New(t)
 
+	olapDriver := os.Getenv("OLAP_DRIVER")
+	if olapDriver == "" {
+		olapDriver = "duckdb"
+	}
+	olapDSN := os.Getenv("OLAP_DSN")
```

Comment:
Seems a little hacky – this should ideally be handled through options in `InstanceOptions` and correct abstractions in the tests.

If this is necessary for now, we should add a prefix like `RILL_RUNTIME_TEST` to these, and get rid of it (in favor of better test abstractions) when refactoring the query abstractions.

Reply:
Makes sense. I can add the prefix for now and refactor tests during the refactoring.